<launch>
  <param name="use_sim_time" type="bool" value="true" />
  <arg name="depth_image_topic" />
  <arg name="camera_matrix_topic" />
  <arg name="camera" default="XTION" />
  <param name="robot_description" command="$(find xacro)/xacro.py '$(find arm_robot_model)/models/DarpaRobot.urdf.xacro'" />
  <!-- Pose Estimation Node-->
  <node name="robot_tracker" type="robot_tracker" pkg="state_filtering" output="screen" respawn="false">
<!--launch-prefix="gdb -ex run args">< launch-prefix="valgrind"-->
    
    <!--param name="tf_correction_root" type="string" value="BASE" /-->
    <param name="robot_description_package_path" type="string" value="$(find arm_robot_model)" />
    <param name="depth_image_topic"    type="string"  value="$(arg depth_image_topic)" />
    <param name="camera_info_topic"  type="string"  value="$(arg camera_info_topic)" /> 
    <!--param name="camera_frame"  type="string"  value="$(arg camera)_RGB" /--> 
    <rosparam command="load" file="$(find state_filtering)/config/robot_config.yaml" />
    <param name="robot_description_package_path" type="string" value="$(find arm_robot_model)" />
  </node>

  <!-- DEBUG -->
  <node pkg="tf" type="static_transform_publisher" name="$(arg camera)_IR_base_link1" args="0 -0.045 0 0 0 0  /$(arg camera) /$(arg camera)_RGB 100" />

</launch>
